app/Http/Requests/Api/Auth/RegisterRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Requests\Api\Auth;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class RegisterRequest extends FormRequest
+{
+    public function rules()
+    {
+        return [
+            'name' => 'required|string|max:255',
+            'email' => 'required|string|email|max:255|unique:users',
+            'password' => 'required|string|min:6|confirmed',
+            'phone_number' => 'string',
+            'dob' => 'date',
+            'profile_img' => 'string',
```

Comment:
I think the image will not be string type, it will be image and please add some other validation like the meme type and the size